### PR TITLE
added session Manager

### PR DESCRIPTION
**Pull request by geoffrey1330**

### CODE REVIEW

File: level2/main.tf [hunk 1]
```diff
@@ -52,9 +45,9 @@ data "aws_ami" "amazonlinux" {
 }
 
 resource "aws_launch_configuration" "web_config" {
-  image_id             = data.aws_ami.amazonlinux.id
-  instance_type        = "t2.micro"
-  key_name             = "main"
+  image_id      = data.aws_ami.amazonlinux.id
+  instance_type = "t2.micro"
+  #key_name             = "main"
```
Comment (vladimirmukhin) [inline level2/main.tf]: Please don't leave commented-out lines. It's a bad style practice

File: level2/iam.tf [hunk 2]
```diff
@@ -24,6 +24,8 @@ resource "aws_iam_policy" "iam_policy_s3" {
  EOF
 }
 
+
+
```
Comment (vladimirmukhin) [inline level2/iam.tf]: Only one newline should be between top-level blocks

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]
Please also remove all security group rules for port 22

File: level2/main.tf (hunk 1 above)
Comment (geoffrey1330) [reply level2/main.tf]: I forget to remove it. will do that now

**Review by vladimirmukhin — APPROVED** [review]

**Pull request merged**